cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
I believe you that any input file runs if all you include is the tag that loads xinclude. This is expected. However, it's still not clear to me that you actually included any external file. It seems from the files here that you just inserted the call to the xinclude library. That is, in the files in this repository (there are many I may have missed it), I didn't see any external xml file being included in a main input file (in place of the recipe, for example).

I'm afraid maybe I'm not communicating the idea clearly.